File: Backend/CarRentalApp/CarRentalDal/Models/Order.cs
namespace CarRentalDal.Models
{
    public class Order
    {
        public int Id { get; set; }

        public Guid ClientId { get; set; }

        public Guid CarId { get; set; }

        public decimal OverallPrice { get; set; }

        public DateTime StartRent { get; set; }

        public DateTime FinishRent { get; set; }

        public User Client { get; set; } = null!;

        public Car Car { get; set; } = null!;

        public ICollection<OrderCarService> OrderCarServices { get; set; } = null!;
    }
}

File: Frontend/CarRentalAdmin/src/app/_models/user/order.ts
import { Car } from '../car/car';
import { CarService } from '../car/car-service';
import { RentalCenter } from '../center/rental-center';

export interface Order {
    overallPrice: number;
    startRent: Date;
    finishRent: Date;
    orderCarServices: CarService[];
    car: Car;
    rentalCenter: RentalCenter;
}